### PR TITLE
CONTRIBUTION Guide - Add README-SPEC.md 

### PR DESCRIPTION
**Pull request by hepiyellow**

### CODE REVIEW

File: docs/contribution/COMPONENT_STRUCTURE.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ wix-style-react
 |  |    ComponentName.scss                  # the component stylesheet
 |  |    ComponentName.spec.js                 # component tests
 |  |    ComponentName.driver.js               # vanilla js public driver to abstract common actions
+|  |    README-SPEC.md                        # Component specification
```
Comment (shlomitc) [inline docs/contribution/COMPONENT_STRUCTURE.md]: maybe just `COMPONENT-SPEC`?

File: docs/contribution/COMPONENT_STRUCTURE.md [hunk 2]
```diff
@@ -53,6 +54,18 @@ wix-style-react
 1. The documentation of every component. It uses the `.story` convention in order to apply automated documentation tool.
 1. Read the [Component Documentation section](./DOCUMENTING_COMPONENTS.md) for more information.
 
+### README-SPEC.md
```
Comment (shlomitc) [inline docs/contribution/COMPONENT_STRUCTURE.md]: worth adding also a boilerplate for this spec

File: docs/contribution/COMPONENT_STRUCTURE.md (hunk 1 above)
Comment (hepiyellow) [reply docs/contribution/COMPONENT_STRUCTURE.md]: ok

Comment (yardnsm) [reply docs/contribution/COMPONENT_STRUCTURE.md]: I think it's better to follow our current `README`s convention, just like `README.TESTKIT.md` and `README.ACCESSIBILITY.md`, we could have `README.SPEC.md`.

If not, maybe using `snake_case` will be better (like `COMPONENT_STRUCTURE.md`), so `COMPONENT_SPEC.md`.

File: docs/contribution/COMPONENT_STRUCTURE.md (hunk 2 above)
Comment (yardnsm) [reply docs/contribution/COMPONENT_STRUCTURE.md]: @mykas has a good one (he used it for `<AddItem/>`).

Comment (hepiyellow) [reply docs/contribution/COMPONENT_STRUCTURE.md]: done

**Pull request merged**